third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -5701,4 +5711,28 @@ AXPlatformNodeWin* AXPlatformNodeWin::GetFirstTextOnlyDescendant() {
   return nullptr;
 }
 
+bool AXPlatformNodeWin::IsDescendantOf(AXPlatformNode* ancestor) const {
+  if (AXPlatformNodeBase::IsDescendantOf(ancestor)) {
+    return true;
+  }
+
+  if (!ancestor) {
+    return false;
+  }
```

Comment:
Does the base `IsDescendantOf` method ever return true for nullptr? If not, then I'd move this above that check since it's cheaper than a function call.